clang/lib/Sema/SemaLookup.cpp

```diff
@@ -1248,6 +1248,13 @@ static DeclContext *findOuterContext(Scope *S) {
   return nullptr;
 }
 
+static bool isDependentAssignmentOperator(DeclarationName Name,
```

Comment:
Ah, perhaps I was confusing... i'd like this inside the `namespace {`, now below.